blivet/udev.py

```diff
@@ -444,18 +444,18 @@ def device_get_devname(info):
     return info.get('DEVNAME')
 
 
-def device_get_slaves(info):
-    """ Return a list of udev device objects representing this device's slaves. """
-    slaves_dir = device_get_sysfs_path(info) + "/slaves/"
+def device_get_parents(info):
+    """ Return a list of udev device objects representing this device's parents. """
+    parents_dir = device_get_sysfs_path(info) + "/slaves/"
```

Comment:
I wonder if the sysfs `slaves` directory will change in the future. That would be a huge API break.